h2o-py/h2o/model/metrics_base.py

```diff
@@ -1817,6 +1822,68 @@ def auuc(self, metric=None):
             assert metric in ['qini', 'lift', 'gain'], \
                "AUUC metric "+metric+" should be 'qini','lift' or 'gain'."
             return self._metric_json['auuc_table'][metric][0]
+
+    def qini(self):
+        """
+        Retrieve Qini value (area between Qini cumulative uplift curve and random curve)
+        
+        :returns: Qini value.
+
+        :examples:
+        
+        >>> from h2o.estimators import H2OUpliftRandomForestEstimator
+        >>> train = h2o.import_file("https://s3.amazonaws.com/h2o-public-test-data/smalldata/uplift/criteo_uplift_13k.csv")
+        >>> treatment_column = "treatment"
+        >>> response_column = "conversion"
+        >>> train[treatment_column] = train[treatment_column].asfactor()
+        >>> train[response_column] = train[response_column].asfactor()
+        >>> predictors = ["f1", "f2", "f3", "f4", "f5", "f6"]
+        >>>
+        >>> uplift_model = H2OUpliftRandomForestEstimator(ntrees=10, 
+        ...                                               max_depth=5,
+        ...                                               treatment_column=treatment_column,
+        ...                                               uplift_metric="qini",
+        ...                                               distribution="bernoulli",
+        ...                                               gainslift_bins=10,
```

Comment:
I gen an error for this field:

__init__() got an unexpected keyword argument 'gainslift_bins'

and when I remove it I got:

Argument `uplift_metric` should be a ?Enum["kl", "euclidean", "chisquared", "auto"], got string qini

Is it only on my site? I tried to rebuild it twice and I have the same problem